app/controllers/course/forum/posts_controller.rb

```diff
@@ -38,6 +37,7 @@ def vote
 
   def destroy
     if super
+      @topic.update_column(:latest_post_at, @topic.posts.last&.created_at || @topic.created_at)
```

Comment:
destroy the posts should not change the topic order, what you think ?

Reply:
I think if we destroy the latest post, we should update this to the next existing latest post. No point seeing the top topic with a very old 'latest post'.

Reply:
I am okay with this, just that I think It would surprise user topic order changes after destroying the post, don't think normal forum does that.